modules/ROOT/pages/salesforce/salesforce-connector-100-studio.adoc

```diff
@@ -63,132 +76,217 @@ IMPORTANT: For the *On New Object* and *On Modified Object* operations, specify
 +
 . For each of the connection choices, you can set optional proxy values:
 +
-** *Host* - Host name of the proxy server.
-** *Port* - The port number the proxy server runs on.
-** *Username* - The username to log in to the server.
-** *Password* - The corresponding password.
+** *Host*
++
+Host name of the proxy server
++
+** *Port*
++
+Port number on which the proxy server runs
++
+** *Username*
++
+Username to log in to the server
++
+** *Password*
++
+Corresponding password
 
-After setting the *Global Element Properties* screen, return to the main connector menu and set the <<Connector Property Values>>.
+After completing fields in the *Global Element Properties* screen, return to the main connector menu and set the <<Connector Property Values>>.
 
 == Basic Authentication
 Important fields:
 
-* *Username* (required): Enter the Salesforce username.
-* *Password* (required): Enter the corresponding password.
-* *Security token*: Enter the corresponding security token.
-* *Authorization URL*: Enter the URL for the endpoint configured to handle SOAP authentication requests.
+* *Username* (required)
++
+Salesforce username
++
+* *Password* (required)
++
+Corresponding password
+* *Security token*
++
+Corresponding security token
++
+* *Authorization URL*
++
+URL for the endpoint configured to handle SOAP authentication requests
```

Comment:
"handle" is another of those terms that can confuse nonnative readers. Consider replacing with "manage" or "process" or similar.